.devcontainer/devcontainer.json

```diff
@@ -21,25 +21,37 @@
 			"installDockerBuildx": true,
 			"version": "latest",
 			"dockerDashComposeVersion": "v2"
+		},
+		"ghcr.io/devcontainers/features/desktop-lite:1": {
```

Comment:
I think we should comment this out and add instructions to the README so that only developers who need it can enable it?